### PR TITLE
Add support for typescript

### PR DESCRIPTION
**Pull request by Guutong**
This PR I have added basic support for TypeScript language.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,7 +29,9 @@
     "husky": "^3.0.0",
     "jest": "^24.8.0",
     "lint-staged": "^9.2.0",
-    "prettier": "^1.18.2"
+    "prettier": "^1.18.2",
+    "standard": "^14.0.2",
```
Comment (PavelPolyakov) [inline package.json]: why is `standard` here?

File: src/__tests__/types/type.test.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import fastify = require('fastify');
```
Comment (PavelPolyakov) [inline src/__tests__/types/type.test.ts]: why do you combine `import` and `require` here?

File: package.json (hunk 1 above)
Comment (Guutong) [reply package.json]: Oh, I'll remove it.

File: src/__tests__/types/type.test.ts (hunk 2 above)
Comment (Guutong) [reply src/__tests__/types/type.test.ts]: ok, I'll change it.
thank you for your feedback.

Comment (Guutong) [reply src/__tests__/types/type.test.ts]: @PavelPolyakov Please review my code again.

**Comment by PavelPolyakov** [comment]
Properly landed in `2.2.1`

**Pull request merged**